### DISCUSSION TITLE
ISO install of SO2.3.2 on XCP-NG - "Unable to open input kickstart file: curl#37 - "Couldn't open file /tmp/part-include""

### DISCUSSION THREAD

**Discussion by krullos** [Security-Onion-Solutions/securityonion#1812] [Unsupported Versions] [answered]
Hey community,

I've had no success in installing Security Onion 2.3.X (same issue occurs with 2.3.0, 2.3.1 and 2.3.2 ISOs so far) from ISO to a VM on my Dell R610 running XCP-NG 8.1.0

VM is set to use the CentOS 7 Template with 8 CPU cores, 12GB RAM , 250GB HDD. Each time it will boot from the ISO, shows the initial screen (to select "Install", "Install with Minimal Graphics" and "Test Media and Install" - each option results in the same outcome - a couple text loading screens followed by a black screen with a single flashing cursor (and will sit like that for hours) - exiting that screen will show:

anaconda 21.48.22.147-1 for CentOS 7 started.
 * installation log files are stored in /tmp during the installation
 * shell is available on TTY2
23:47:09 Running pre-installation scripts
The following problem occurred on line 0 of the kickstart file:

Unable to open input kickstart file: curl#37 - "Couldn't open file /tmp/part-include"

Pane is dead

![1](https://user-images.githubusercontent.com/47314874/98055596-b8450b80-1ea2-11eb-8846-9e048f5c59c7.JPG)

**Comment by TOoSmOotH** [comment]
Typically that means it is not seeing any disks. I am not familiar with XCP-NG. Best bet would be to do a centos minimal install and run a lsblk.

**Comment by krullos** [comment, marked as answer]
Thanks for the very quick response, Mike.

Here's what CentOS Minimal install and a lsblk displays on the same VM:

![2](https://user-images.githubusercontent.com/47314874/98059401-ea0ea000-1eab-11eb-9898-be5f723fd966.JPG)

**Reply by TOoSmOotH** [reply]
I opened #1819. Will need to modify kickstart for these drive types.

**Reply by krullos** [reply]
Issue still present with Security Onion 2.3.21 ISO

**Reply by TOoSmOotH** [reply]
I pushed a change to the kickstart script that should help this. It will be out in 2.3.30
